packages/ui-components/src/components/PortalProvider/PortalProvider.stories.tsx

```diff
@@ -5,9 +5,17 @@
 
 import React from "react"
 import { createPortal } from "react-dom"
-import PropTypes from "prop-types"
-import { PortalProvider, usePortalRef } from "."
-import { Message } from "../Message/"
+import { PortalProvider, usePortalRef, PortalProviderProps } from "./PortalProvider.component"
+
+//import { Message } from "../Message" // TODO: use Message after it is available in TS
```

Comment:
Lets wait until message component merge then we actualize it

Reply:
I changed to Button as discussed during the meeting.